test/lambda-handler/index.py

```diff
@@ -3,4 +3,4 @@
 def handler(event, context):
   subprocess.check_call(["/opt/kubectl/kubectl", "config", "view"])
   subprocess.check_call(["/opt/helm/helm", "version"])
-  return
+  return
```

Comment:
Revert?

Reply:
extra line is the culprit of all the other asset changes, i bet...